sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpTransactionEnlistment.cs

```diff
@@ -49,6 +52,21 @@ protected override async Task<AmqpTransactionEnlistment> OnCreateAsync(TimeSpan
             }
         }
 
+        private async Task<Controller> GetController(TimeSpan timeout)
+        {
+            FaultTolerantAmqpObject<Controller> faultTolerantController;
+            if (_transactionGroup == null)
```

Comment:
This isn't thread-safe.   Does that matter here or is a this a benign race?

Reply:
_transactionGroup is only set in the ctor so it doesn't need to be under lock.

Reply:
Nevermind.   This is only set in the constructor, so there's no chance that `_transactionGroup` gets set after the null  check.    Nothing to see here.